test/box/cfg.result

```diff
@@ -263,10 +263,6 @@ box.cfg{sherlock = 'holmes'}
  | ...
 
 -- check that cfg with unexpected type of parameter fails
-box.cfg{listen = {}}
```

Comment:
Why did you remove this case?

Reply:
No it's ok to pass table for listen option! Empty table same as empty listen string.

Reply:
Ok I see this case in the new test file